### ISSUE TITLE
Can one reproduce an existing integration settings from UI?

### ISSUE THREAD

**Issue by mastepan** [syndesisio/syndesis#1441]
Why I am asking:
(I'm referring to TP3 Syndesis app state)

It can be reproducible if we can view used connection's settings.

but...it works this way now:
- If I create an integration and then change any used connection's settings, the changes don't apply to the existing integration. That means the connection used in the integration and the one available for the new integrations can be absolutely different ones.

- therefore I never know what the used  connection's settings are (despite it has got the same name as in the integration)

- thus... can't we disallow editing used connections? Or am I missing any usecase?

- that would mean creating new connections for any new setting that may be used in an integration. And that sound right to me. What do you say?

side note: I know I can export an integration in json and theoretically I can investigate all settings from there. But who would do that? 

To sum it up:
My proposal is to let user edit a connection as long as it ins't being used in an integration. Then disallow editing (and deleting) the connection as it is being used in an integration.

**Comment by zregvart** [comment]
Is this part of #971?

cc @kcbabo @gashcrumb @rhuss

**Comment by mastepan** [comment]
> Is this part of #971?

Might be but haven't found the behavior I had described literally specified anywhere yet.
Could you @zregvart link this to the #971 issue if it make any sense? I'm not sure what workflow you follow. Thanks.

**Comment by kcbabo** [comment]
I agree it would be useful to:

- have some indication that the connection configuration used by an integration is not the same as the current connection config.
- be able to view the configuration used by an integration

I don't think we should force the user to create multiple connections to satisfy the above.  Syndesis should keep track of the connection config used by an integration and provide some way to view that.

**Comment by kcbabo** [comment]
BTW, what I described above does not seem in scope for TP4 to me.  Unless it's an easy fix, then I say ship it! :-)

**Comment by zregvart** [comment]
From what I understand we have this with the integration bulletin board.
